site/app.js

```diff
@@ -56,7 +63,8 @@ if (SP_PROTECTED === "false") {
       done(null, {
         nameID: profile.nameID,
         nameIDFormat: profile.nameIDFormat,
-        email: profile.email
+        email: profile.email,
+        secret: createValidSecret ()
```

Comment:
As discussed - I believe token signature gives us the assurance that token has been securely generated, so we don't need to encrypt date with that too.

Reply:
Ok, have removed it now